universal-application-tool-0.0.1/app/auth/ProfileFactory.java

```diff
@@ -1,7 +1,22 @@
 package auth;
 
+import com.google.common.base.Preconditions;
+import java.time.Clock;
+import javax.inject.Inject;
+
 public class ProfileFactory {
+
+  private Clock clock;
+
+  @Inject
+  public ProfileFactory(Clock clock) {
+    this.clock = Preconditions.checkNotNull(clock);
+  }
+
   public UATProfile createNewApplicant() {
-    return new UATProfile();
+    UATProfile p = new UATProfile(clock);
```

Comment:
nit: rename `UATProfile` to `UatProfile` per [style guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)

Reply:
Done.